addons/escoria-core/game/core-scripts/esc/types/esc_object.gd

```diff
@@ -97,6 +97,9 @@ func _get_interactive() -> bool:
 func _set_interactive(value: bool):
 	if "is_interactive" in self.node:
 		self.node.is_interactive = value
+		if not value:
+			(escoria.game_scene.tooltip_node as ESCTooltip).clear()
```

Comment:
This is dangerous if game scene has no tooltip. See if there is a method to call in ESCGame for this instead